### PR TITLE
Add fields for the in-progress SAK-44055

### PR DESCRIPTION
**Pull request by csev**
This will allow this feature to be added to a 21.x release

@bjones86 I need some Oracle help.  Should be pretty straightforward.

### CODE REVIEW

File: docs/conversion/sakai_21_mysql_conversion.sql [hunk 1]
```diff
@@ -19,3 +19,8 @@ ALTER TABLE rbc_criterion ADD COLUMN WEIGHT DOUBLE NULL DEFAULT 0;
 -- Existing records needed to be switched on right before the feature is used
 UPDATE lti_tools SET pl_lessonsselection = 1;
 -- END SAK-44637
+
+-- SAK-44055 - IMS LTI Advantage Autoprovision
+ALTER TABLE lti_tools ADD lti13_auto_token VARCHAR(1024);
+ALTER TABLE lti_tools ADD lti13_auto_state INT;
+ALTER TABLE lti_tools ADD lti13_auto_registration MEDIUMTEXT;
```
Comment (bjones86) [inline docs/conversion/sakai_21_mysql_conversion.sql:26]: @csev, do you know if this field (`lti13_auto_registration`) will exceed 4,000 characters?

Comment (bjones86) [inline docs/conversion/sakai_21_mysql_conversion.sql:26]: If `lti13_auto_registration` will not exceed 4,000 characters:

```
ALTER TABLE lti_tools ADD lti13_auto_token VARCHAR2(1024);
ALTER TABLE lti_tools ADD lti13_auto_state NUMBER(10,0);
ALTER TABLE lti_tools ADD lti13_auto_registration VARCHAR2(4000);
```

If `lti13_auto_registration` *will* exceed 4,000 characters:

```
ALTER TABLE lti_tools ADD lti13_auto_token VARCHAR2(1024);
ALTER TABLE lti_tools ADD lti13_auto_state NUMBER(10,0);
ALTER TABLE lti_tools ADD lti13_auto_registration CLOB;
```

**Review by bjones86 — CHANGES_REQUESTED** [review]

File: docs/conversion/sakai_21_mysql_conversion.sql (hunk 1 above)
Comment (ottenhoff) [reply docs/conversion/sakai_21_mysql_conversion.sql:26]: @csev ?

**Comment by ern** [comment]
Closing, already incorporated.

**Pull request closed without merging**